tests/avocado/avocado_qemu/test.py

```diff
@@ -245,12 +246,15 @@ def get_console(self, console_address=None, prompt=r"[\#\$] "):
         :param prompt: The regex to identify we reached the prompt.
         """
 
+        if not all((self.username, self.password)):
+            raise QEMULoginError('Username or password not set.')
+
         if not self.is_running():
-            raise QEMUConsoleError('VM is not running.')
+            raise QEMULoginError('VM is not running.')
 
         if console_address is None:
             if self._console_address is None:
-                raise QEMUConsoleError("Can't determine the console address "
+                raise QEMULoginError("Can't determine the console address "
```

Comment:
... but not this! Sorry if I wasn't clear enoug.  I do think this situation here is indeed about the console (device, address, access), while the previous one is really about authentication (username/password).

Reply:
oh, ok , sorry. adjusting